scripts/tooling/logging.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2025 Tenstorrent AI ULC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LOGGING_H_
+#define LOGGING_H_
```

Comment:
Is it possible to use more unique header guards such as `TENSTORRENT_TOOLING_LOGGING_H_`? There is no namespacing at all here.

Same with `vuart.h`